src/components/Plans/UpdatePaymentInformation/UpdatePaymentInformation/Reprocess/index.js

```diff
@@ -0,0 +1,135 @@
+import React, { useEffect, useState } from 'react';
+import { InjectAppServices } from '../../../../../services/pure-di';
+import { useIntl, FormattedDate, FormattedNumber, FormattedMessage } from 'react-intl';
+import { Loading } from '../../../../Loading/Loading';
+import { handleMessage } from '../index';
+import { StatusMessage } from '../../../Checkout/PurchaseSummary/PlanPurchase/index';
+import useTimeout from '../../../../../hooks/useTimeout';
+
+const dollarSymbol = 'US$';
+const HAS_ERROR = 'HAS_ERROR';
+const SAVED = 'SAVED';
+export const DELAY_BEFORE_REDIRECT_TO_SUMMARY = 3000;
+
+const numberFormatOptions = {
+  style: 'decimal',
+  minimumFractionDigits: 2,
+  maximumFractionDigits: 2,
+};
+
+const DeclinedInvoices = ({ declinedInvoices }) => {
+  const intl = useIntl();
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  return (
+    <>
+      <table className="dp-c-table  m-t-24">
+        <thead>
+          <tr>
+            <th>
+              <a href="#">
+                {_('updatePaymentMethod.reprocess.invoices_declined_table.date_column')}
+              </a>
```

Comment:
We are trying to use buttons styled like links in place of `a` when they are not real links.

Reply:
I will remove the ```a``` element. Becuase only we need to show the title in the header.